babeld/message.c

```diff
@@ -144,7 +144,12 @@ static bool parse_update_subtlv(const unsigned char *a, int alen,
 		       "Received Mandatory bit set but this FRR version is not prepared to handle it at this point");
 		return true;
 	} else if (type == SUBTLV_PADN) {
-		/* Nothing. */
+        for (int j = 0; j < len; j++) {
+            if (a[i + j + 2] != 0) {
```

Comment:
Please no magic values.  Please go through the entire commit and fix the magic values to a define.